src/view/Cart/components/Order/style.ts

```diff
@@ -3,6 +3,19 @@ import { CounterWrapper } from '@components/Counter/style'
 import { Paragraph, CardPrice, HeadlineFour } from '@components/Typography'
 import { devices } from '@constants/devices'
 
+export const DeleteButton = styled.button`
+  background: transparent;
+  border: none;
+  color: #a4351d;
```

Comment:
Salve grande mestre Francis! 🎖️ 
O que você acha de alterar essa cor em hex pela nossa constante de cor?

```suggestion
  color: ${({theme}) => theme.colors.red.main };
```